README.md

```diff
@@ -1,12 +1,43 @@
+
 [![Join the chat at https://gitter.im/fetch-amul-topicals/Lobby](https://badges.gitter.im/fetch-amul-topicals/Lobby.svg)](https://gitter.im/fetch-amul-topicals/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-A Python script that fetches details all ‘topicals’ across all years from the Amul website (http://www.amul.com/m/amul-hits) and has the feature to download the pictures by year into your local system
+It uses a Python script (aka **web scraper**) that fetches details (here ‘topicals’) across all years from the Amul's website (http://www.amul.com/m/amul-hits) and has the feature to download the pictures chronologically into your local system.
+
+### Fundamentals of Web Scraping
+Web scraping is a computer software technique of extracting information from websites. This technique mostly focuses on the transformation of unstructured data (HTML format) on the web into structured data (database or spreadsheet). <br>
+Tips:
+Get familiar with the HTML tags ; CSS, Layout & Architecture to extract the data in a better manner
+
+### Steps
```

Comment:
Either we should expand on the steps of installation and usage, or don't use this at all. This 'source code reading' part is understood by reading the code.

Reply:
Chuck your steps 1-5. No need.